demo/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "react-accordion-demo",
+  "version": "1.0.0",
+  "description": "A really lightweight, symantic, super simple accordion for React",
+  "type": "module",
+  "scripts": {
+    "develop": "servatron -d dist & esbuild --watch --bundle src/index.tsx --outfile=dist/index.min.js --loader:.js=jsx",
```

Comment:
Does this watch both the demo code AND the accordion code?

Reply:
Nope. Demo would be a separate self contained project.